config/settings.yml

```diff
@@ -2,11 +2,16 @@ hours_api: 'https://library-hours.stanford.edu/'
 
 featured_items:
   - image_url: https://stacks.stanford.edu/image/iiif/qy843rv0944/qy843rv0944_0080/200,700,2000,2500/!800,800/0/default.jpg
-    caption: USA Exhibit 388, Taube Archive of the International Military Tribunal (IMT) at Nuremberg, 1945‑46
+    image_alt: USA Exhibit 388, Taube Archive of the International Military Tribunal (IMT) at Nuremberg, 1945-46
+    caption_html: USA Exhibit 388, Taube Archive of the International Military Tribunal (IMT) at Nuremberg, 1945-46
   - image_url: https://stacks.stanford.edu/image/iiif/gs873gr6640/gs873gr6640_0001/full/!800,800/0/default.jpg
-    caption: Crucifix, Fear, Lynn Beldner Punk Music Photograph Collection
+    image_alt: Crucifix, Fear, from Lynn Beldner Punk Music Photograph Collection
+    caption_html: |
+      Crucifix, Fear, from <a href="/catalog/ars0213aspace_dadf95f978f9454e903d382f04dd299c">
+      Lynn Beldner Punk Music Photograph Collection</a>
```

Comment:
Hey @taylor-steve -- so this link doesn't work locally for me, because this item isn't indexed. I think we should account for that by either adding this EAD to `spec/fixtures`, adding an example link from one of the fixture EADs instead, or simply leaving out the example link for now. Thoughts?

Reply:
Yeah @marlo-longley that makes total sense, thanks. I'll take a look at what we have and do one or the other.

Reply:
I took out the link for now. I can't find that item in ASpace and it's surprisingly hard to find something with suitable image.

Reply:
Cool. I think that makes total sense.